site/src/components/sampler/largestTriangleThreeBucket.md

```diff
@@ -47,7 +47,14 @@ example-code: |
 
 The Largest Triangle Three Bucket sampling component is a method of subsampling data to improve performance with large data sets. The algorithm entails calculating the largest areas between a data point and its neighbouring buckets.
 
-The sampler requires both the `xValue` and `yValue` properties in order to calculate the area. You can configure the sampling frequency by setting the `bucketSize` property.
+The sampler requires both the `x` and `y` properties in order to calculate the area. You can configure the sampling frequency by setting the `bucketSize` property. If you're dealing with irregularly spaced data, you can call the respective scales on the accessor functions to ensure the area calculated is more representative of the data being displayed, for example:
```

Comment:
> If you're dealing with irregularly spaced data, you can call the respective scales on the accessor functions to ensure the area calculated is more representative of the data being displayed, for example:

This doesn't sound right to me. I thought the issue was that the algorithm assumed that the x and y scales are continuous and linear? I'd suggest the following:

"As the  The Largest Triangle Three Bucket sampler computes areas, if you have a non-linear or discontinuous scale, the values supplied to the sampler must be scaled accordingly. You can apply the respective scales in the accessor functions as follows:"